urshbase.sql

```diff
@@ -0,0 +1,21 @@
+CREATE DATABASE urshbase;
+USE urshbase;
+
+CREATE TABLE shorten_urls (
+id int(11) NOT NULL AUTO_INCREMENT,
+code varchar(5) NOT NULL,
+point varchar(256) NOT NULL,
+PRIMARY KEY (id)
+);
+
+INSERT INTO shorten_urls ('code', 'point') VALUES 
```

Comment:
Please remove this, it is actually unnecessary because we need a clear database structure.